### ISSUE TITLE
Weird white visual effect on the right side of text areas

### ISSUE THREAD

**Issue by FalcoGer** [eranif/codelite#2483]
I just upgraded to ubuntu mate 20.04 (from 18.04) and suddenly codelite has this weird white overlay to the right of all text fields. This is especially annoying and distracting in the dark layout. I haven't found any reference on how to turn it off, so I assume it's a bug.

https://ibb.co/McLFRcS

**Comment by eranif** [comment]
Its a bug with wxGTK3 & Ambiance theme (I think that's the name of the default theme).
I use Adwaita dark and it's hardly notice. But I can't really do anything about this (its not CodeLite source code, but rather in one of it's dependencies)

**Comment by FalcoGer** [comment]
I have found out that it's the effect for visualizing that the user has scrolled off the end of a scrollable element. If you write past the end of the visible portion of the text field, the effect starts to lessen and eventually is removed.
curiously this effect doesn't pop up when I scroll down past the end of the file.

Curiously this problem occurs in codelite with any style selected (not just Ambiance as you claim or Monokai_2, which I use)
Also curiously in pluma, which produces the effect when scrolling past the end with the mouse wheel, it works perfectly well with any of it's themes selected.

I also just upgraded wxWidgets to 3.1.4 and the problem still persists.

So what is the problem here? Where do I report this?
